values.yaml

```diff
@@ -343,6 +343,21 @@ server:
   # `server.ports.serflan.port` must be set to something other than 8301.
   exposeGossipAndRPCPorts: false
 
+  # [Enterprise Only] Configures the partitions and the port for each partition.
+  # Accepts an array of map with field name and port where where the name is the
+  # name of the partition and port is the desired gossip port for that partition.
+  # Example:
+  #
+  # ```yaml
+  # partitions:
+  # - name: foo
+  #   port: 8503
```

Comment:
Curious if this is the gossipPort would we want to call it that instead of `port`?  🤷🏽